tests/integration_tests/performance/test_snapshot_restore_performance.py

```diff
@@ -186,41 +186,21 @@ def consume_output(cons, result):
 
 
 @pytest.mark.nonci
-@pytest.mark.parametrize("vcpu_count", [1, 2, 4, 8, 10])
-def test_snapshot_scaling_vcpus(
-    bin_cloner_path, microvm_factory, rootfs, guest_kernel, vcpu_count, st_core
+@pytest.mark.parametrize(
+    "mem, vcpus",
+    [(128, 1), (2048, 2), (4096, 3), (6144, 4), (8192, 5), (10240, 6), (12288, 7)],
```

Comment:
Would it make sense to have some intermediate ones like `(1024, 1)`, to make it obvious that memory dominates, and it's not the CPUs?

Reply:
Yeah, I think maybe having a few more scenarios at the lower end of things might make sense, given that "microvm" is our tagline. I'll add (1024, 1)

Reply:
A comment explaining where these values come from could be helpful in case we need to update them.